WooCommerce/build.gradle

```
@@ -346,7 +349,7 @@ android.buildTypes.all { buildType ->
     }
 }
 
-task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.gradle.ViolationCommentsToGitHubTask) {
+task violationCommentsToGitHub(type: ViolationCommentsToGitHubTask) {
```

```suggestion
tasks.register("violationCommentsToGitHub", ViolationCommentsToGitHubTask) {
```

Please see my other comment for context about this suggestion.